index.html

```diff
@@ -18,39 +18,39 @@
         <div class="logoPlaceHolder">
           <a href="#" class="logo"> My Logo</a>
         </div>
-        <ul id="menu">
+        <ul class="menu">
           <li class="Menu-portfolio"><a href="#">Portfolio</a></li>
           <li class="Menu-about"><a href="#About">About</a></li>
           <li class="Menu-contact"><a href="#Contact">Contact</a></li>
         </ul>
-        <div id="tools-bar">
-          <div id="Hamburger">
-            <div class="union">
-              <span class="bar"></span>
-              <span class="bar"></span>
-              <span class="bar"></span>
-            </div>
+        <div id="Hamburger">
+          <div class="union">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
           </div>
         </div>
+        
       </nav>
     </header>
     <div class="Headline-section">
       <h1 class="Headline-section-title">
         I'm Adeyemi Glad to see you!
       </h1>
-      <p class="Headline-section-body">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-    </p>
-    <div class="social-section">
-      <h2 class="social-connect-section">LET'S CONNECT</h2>
-    </div>
-    <div class="social-section-icon"> <!-- review-->
-      <ul class="icons">
-        <li><a href="twitter.com/Mc_Emarkees" class="fa-brands fa-twitter"></a></li>
-        <li><a href="github.com/emarkees" class="fa-brands fa-github"></a></li>
-        <li><a href="linkedin.com" class="fa-brands fa-linkedin-in"></a></li>
-        <li><a href="medium.com" class="fa-brands fa-medium"></a></li>
-        <li><a href="angellist.com" class="fa-brands fa-angellist"></a></li>
-      </ul>
```

Comment:
You have done very well on this styling. However, since we have to make the project look exactly like the design template, I suggest that you change the color of the social media icons, found in the headline section and the about-myself section so as to look exactly like the template. 
Your project: 
![image](https://user-images.githubusercontent.com/107795045/222868424-64b21ea8-8d47-4d7d-a4eb-b6540d1d94c8.png)

Figma design: 
![image](https://user-images.githubusercontent.com/107795045/222868489-e0b9b9d2-901b-4e02-af65-80fd851bc84c.png)